client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1991,6 +1991,21 @@ Undocumented.prototype.getGuidedTransferStatus = function( siteId ) {
 };
 
 /**
+ * Requests the status of a guided transfer
+ *
+ * @param {int} siteId  The site ID
+ * @returns {Promise} Resolves to the response containing the transfer status
+ */
+Undocumented.prototype.saveGuidedTransferHostDetails = function( siteId, hostDetails ) {
```

Comment:
So I'm thinking this might be the wrong place for this function. I think to be consistent with how things are done for export we should do one of a few things here:
1. create a file in [client/lib/wpcom-undocumented/lib](https://github.com/Automattic/wp-calypso/tree/master/client/lib/wpcom-undocumented/lib) called `guided-transfer.js` and wire it up in the same way as `export.js` and then use it by calling `wpcom.undocumented().guidedTransfer( guidedTransferId ).saveHostDetails(...)`
2. Add the `saveGuidedTransferHostDetails`to the [export lib](https://github.com/Automattic/wp-calypso/blob/master/client/lib/wpcom-undocumented/lib/export.js) and call it like `wpcom.undocumented().export(<Note that theres no ID here>).saveGuidedTransferHostDetails(...)`
3. Some other way 😄